editor/components/block-list/style.scss

```diff
@@ -334,23 +326,37 @@
 			border-bottom: 3px solid $blue-medium-500;
 		}
 	}
+}
 
 
-	/**
-	 * Left and right side UI
-	 */
+/**
+ * Left and right side UI
+ */
+
+.editor-block-list__block {
 
+	// Left and right
 	> .editor-block-settings-menu,
 	> .editor-block-mover {
 		position: absolute;
-		top: 0;
+		top: -.9px; // .9px because of the collapsing margins hack, see line 27, @todo revisit when we allow margins to collapse
+		bottom: -.9px; // utilize full vertical space to increase hoverable area
 		padding: 0;
+		width: $block-side-ui-width;
+	}
+
+	// Elevate when selected or hovered
+	&.is-selected,
+	&.is-hovered {
+		.editor-block-settings-menu,
+		.editor-block-mover {
+			z-index: 9999;	// @todo
```

Comment:
Was this planned to be addressed here?

Reply:
Ack yes that was intended. All it needs is a proper z index with the mixin. I'm out right now but can look later, alternately feel free to push a fix if you have bandwidth.

Reply:
There's a `'.editor-block-mover': 1` z-index as well. Is that one made redundant with this? Can the controls always have a high z-index, even without the `is-selected` / `is-hovered` qualifiers? Are they even present when one of those conditions doesn't apply?

In meantime, I've pushed e1336d3 with reference to `z-index` list here.

Reply:
I believe that as I worked on this, I decided that yes, the side UI can always be above other things, except in the mobile configuration. So I'd think the old one is redundant now.